.python-version

```diff
@@ -1,4 +0,0 @@
-2.7.18
-3.6.9
-3.7.8
-3.8.5
```

Comment:
I removed the `.python-versions` file because it was forcing pyenv to want these specific patch versions when testing. I had 3.6, 3.7 and 3.8 installed, but not on the patch version listed here and it wouldn't use them. Instead of locking to a particular patch version it seemed like locally setting up a pyenv file might work. Open to other suggestions though if there is a better way to handle this.

Reply:
Yes, I think we can remove it.